### DISCUSSION TITLE
Updating the Teams.js version referenced by this library

### DISCUSSION THREAD

**Discussion by yuukitakeuchi** [OfficeDev/microsoft-365-agents-toolkit#11398] [Q&A] [answered]
I’ve come across information indicating that due to upcoming domain changes in Microsoft services, there is a requirement to update Teams.js to version 2.19 or later by June 2024. 

I am currently developing a Teams app using the Teams Toolkit template in Visual Studio 2022. Upon inspecting the generated code, I noticed it references Teams.js version 2.17. I would like to update this to version 2.19 or above. 

Could anyone guide me on how to proceed with this update?

[Microsoft teams.js docs](https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?tabs=manifest-teams-toolkit#configure-content-security-policy-headers)

**Comment by therealjohn** [comment, marked as answer]
We plan to update the Teams.js version to 2.19 in the next pre-release.
